README.md

```diff
@@ -6,8 +6,11 @@
 	<img src="http://elucidata.io/static/images/elucidata-logo-with-text.svg">
   </a>
 
-## Travis Build
-[![Build Status](https://travis-ci.org/ElucidataInc/ElMaven.svg?branch=develop)](https://travis-ci.org/ElucidataInc/ElMaven)
+## Build Status
+[![Build Status](https://travis-ci.org/ElucidataInc/ElMaven.svg?branch=ElMaven-CI)](https://travis-ci.org/ElucidataInc/ElMaven)
```

Comment:
@rish9511 I can't tell just by looking at the build status what they mean. The travis one does not even have an icon. Can we show these separately for Windows, Ubuntu and Mac OS?

Reply:
@rish9511 I was wondering if we could show the build status for each platform separately. Something like: Linux|passing MacOS|passing Windows|passing

I understand that we don't have separate build status for Linux and MacOS so this might not be possible. I'll resolve this conversation.